eglot-x.el

```diff
@@ -207,6 +207,15 @@ server detects a problem, this extension makes the beginning of
 the debugging process a tiny bit easier."
   :type 'boolean)
 
+(defcustom eglot-x-enable-inactive-code t
+  "If non-nil, enable inactive code notifications support. This
+deemphasizes code removed by the preprocessor based on
+compile-time information."
+  :type 'boolean
+  :link '(url-link
+          :tag "inactive code explanation (ccls)"
+          "https://joaotavora.github.io/eglot/#Extending-Eglot-1"))
```

Comment:
The tag says "ccls", but the linked text talks about clangd. 

I think it would be better to use the links added to the README.